hazelcast/src/main/java/com/hazelcast/map/impl/event/AbstractFilteringStrategy.java

```diff
@@ -83,7 +83,8 @@ protected boolean evaluateQueryEventFilter(EventFilter filter, Data dataKey, Obj
      */
     private Extractors getExtractorsForMapName(String mapNameOrNull) {
         if (mapNameOrNull == null) {
-            return Extractors.empty();
+            return Extractors.newBuilder(((InternalSerializationService) serializationService))
```

Comment:
Can you change serializationService to be of type internalSerializationService so we don't need to do this cast all over the place?

Reply:
done

Reply:
the cast is still here :)

Reply:
fixed